api/services/elastigroup/aws/schemas/elastigroup-scale-down.yaml

```diff
@@ -0,0 +1,120 @@
+type: object
+title: scale down config
+properties:
+  policyName:
+    type: string
+    description: |
+      The policy name
+    example: "my-down-policy"
+  metricName:
+    type: string
+    description: |
+      The name of the metric.
+    default: "CPUUtilization"
+    example: "CPUUtilization"
+  statistic:
+    type: string
+    description: |
+      The metric statistics to return.
+    enum: [ average , sum , sampleCount , maximum , minimum , percentile ]
+    example: "average"
+  extendedStatistic:
+    type: string
+    description: |
+      Percentile statistic. Valid values: p0.1 - p100
+    example: "p1.5"
+  unit:
+    type: string
+    description: |
+      The unit for the alarm's associated metric.
+    enum: [ seconds , microseconds , milliseconds , bytes , kilobytes , megabytes , gigabytes , terabytes , bits , kilobits , megabits , gigabits , terabits , percent , count , bytes/second , kilobytes/second , megabytes/second , gigabytes/second , terabytes/second , bits/second , kilobits/second , megabits/second , gigabits/second , terabits/second , count/second , none ]
+    example: "percent"
+  threshold:
+    type: integer
+    description: |
+      The value against which the specified statistic is compared
+    example: 90
+  action:
+    type: object
+    example: "adjustment"
```

Comment:
This is an example for the whole action object..? it is wrongly displayed on the client